lib/3scale/backend/transactor.rb

```diff
@@ -182,6 +188,24 @@ def preload_usage_limits(application, no_body_enabled, usage_in_params)
         end
       end
 
+      def filter_usages_with_0(transactions)
+        res = transactions.clone
+
+        # There are plenty of existing tests using both a string and a symbol
+        # when accessing the usage.
+        usage_key = ['usage'.freeze, :usage]
```

Comment:
`params` is an `IndifferentHash` when set up by Sinatra, but as the comment says, we have _plenty_ of tests calling this method directly and using both options with standard ruby Hashes.

Reply:
Because the tests don't send an IndifferentHash and changing all of them looks like a lot of work that maybe we should do but it's not related to this PR.

Reply:
Ok.